index.html

```diff
@@ -2601,17 +2626,498 @@ <h4>Derived Proof</h4>
+        <section>
+          <h4>Pseuonym with Hidden PID Feature</h4>
+          <section>
+            <h5>Hidden PID Commitment Generation</h5>
+            <p>
+The first step in using the <em>pseudonym with hidden PID</em> feature is for the
+holder to generate its secret |pid| value and then compute a commitment with
+proof for this value according to the <em>commitment computation</em> procedure
+of [[CFRG-Blind-BBS-Signature]]. Example values and outputs of this procedure
+are shown below.
+            </p>
+            <pre class="example nohighlight" title="Hidden PID"
+            data-include="TestVectors/FeatureInputs/hiddenPid.json"
+            data-include-format="text"></pre>
+            <pre class="example nohighlight"
+            title="Hidden PID Commitment Information"
+            data-include="TestVectors/PseudoHiddenPid/commitmentInfo.json"
+            data-include-format="text"></pre>
+            <p>
+The |pid| and |secretProverBlind| are retained by the holder and kept
+secret. The |commitmentWithProof| value needs to be communicated to the issuer.
+            </p>
+          </section>
+          <section>
+            <h5>Hidden PID Base Proof</h5>
+            <p>
+The addition of a base proof under the pseudonym with hidden PID option begins
+with the issuer receiving a |commitmentWithProof| value from the holder and then
+verifying that value with the <em>commitment verification</em> procedure of
+[[CFRG-Blind-BBS-Signature]].
+This example will make use of the same key material shown in
+<a href="#example-private-and-public-keys-for-signature-0"></a>. It will utilize
+the same unsigned document shown in
+<a href="#example-credential-without-proof-for-features"></a> and the same
+mandatory pointers shown in
+<a href="#example-mandatory-pointers-for-features"></a>. This results in the
+same canonical document as in
+<a href="#example-canonical-document-for-features"></a>,
+the same canonical HMAC document as in
+<a href="#example-canonical-hmac-document-for-features"></a>, and the same "add
+base transformation" as in
+<a href="#example-add-base-transformation-for-features"></a>.
+          </p>
+          <p>
+This example makes use of the same proof configuration as in
+<a href="#example-base-proof-configuration-for-features"></a>.
+This results in the same canonical base proof as in
+<a href="#example-canonical-base-proof-configuration-for-features"></a>.
+Combining this with the above assumptions leads to the same base hashes as in
+<a href="#example-add-base-hashes-for-features"></a>.
+          </p>
+          <p>
+Since |featureOption| is equal to `"pseudonym_hidden_pid"` the procedure of
+section <a href="#base-proof-serialization-bbs-2023"></a> will produce the
+output shown below. This makes use of the
+signature generation algorithm of [[CFRG-Pseudonym-BBS-Signature]]. Note the
+inclusion of the |featureOption| as well as the |signerBlind| value as this
+needs to be communicated to the holder.
```

Comment:
```suggestion
Since |featureOption| is equal to `"pseudonym_hidden_pid"`, the procedure of
section <a href="#base-proof-serialization-bbs-2023"></a> will produce the
output shown below. This makes use of the
signature generation algorithm of [[CFRG-Pseudonym-BBS-Signature]]. Note the
inclusion of the |featureOption| as well as the |signerBlind| value, as these
need to be communicated to the holder.
```